formapp/templates/message.html

```diff
@@ -1,36 +0,0 @@
-{% extends 'header.html' %}
```

Comment:
Why was this deleted?

Reply:
Information that is important to the average user will be displayed as an alert. This works better in Advanced mode than the old system.
Other information will be returned in a string. A benefit of doing it this way is that the string can easily be used in other pages.